projects/demo/src/modules/components/input-copy/input-copy.template.html

```diff
@@ -22,6 +22,7 @@
                 [tuiTextfieldLabelOutside]="labelOutside"
                 [tuiTextfieldSize]="size"
                 [tuiTextfieldMaxLength]="maxLength"
+                [tuiTextfieldCleaner]="cleaner"
```

Comment:
I wanted to clarify 😊  do you write e2e tests for such things?

Reply:
We don't write tests for demo portal, we only use demo portal to test components. I believe one of these bugs was found exactly because of deep tests that go over all properties.